### DISCUSSION TITLE
Welcome to linvo-scraper Discussions!

### DISCUSSION THREAD

**Discussion by nevo-david** [linvo-io/linvo-scraper#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kokoali-bima** [comment]
Hi,
Sorry I am so new with these things.
Could you inform me how to run or start this apps?
I am basic on nodejs programming. Hopefully you can give me a little clue how to starting or running it after installation.
What should I do next? So I can use this apps.

**Reply by kokoali-bima** [reply]
FYI, I got PIN requested Notification on my email after running this app.
Where should I put the PIN?
